kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp

```diff
@@ -353,9 +354,10 @@ struct VerifyBlockRequests {
       out << "\t\t\"signature_digest\": \"" << hex_digest << "\",\n";
       out << "\t\t\"persistency_type\": \"" << persistencyType(req.requestPersistencyType) << "\",\n";
       std::string verification_result;
-      auto verifier = std::make_unique<concord::util::crypto::RSAVerifier>(
+      const auto verificationKey = getByteArrayKeyClass<EdDSAPublicKey, EdDSAPublicKeyByteSize>(
```

Comment:
This can be an RSA key. We need to store some metadata like key algo along with the key.